sdk/dataprotection/Microsoft.Azure.Management.DataProtection/src/Microsoft.Azure.Management.DataProtection.Backup.csproj

```diff
@@ -6,13 +6,13 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataProtection.Backup</PackageId>
 		<Description>Provides developers with libraries for data protection under Azure Resource manager.</Description>
-		<Version>1.0.0-beta.1</Version>
+		<Version>1.0.1-beta.1</Version>
```

Comment:
There are a few breaking changes in the API. We need to bump the version to `2.0.0` or at least `1.1.0` (if the SDK is still in beta) with some warnings in the change logs.